src/index.js

```diff
@@ -0,0 +1,209 @@
+
+const extendStringClass = {
+  /**
+   * Returns true if the string contains vowels.
+   * @return {Boolean}
+   */
+
+  hasVowels() {
+    return /[aeiou]/i.test(this);
+  },
+
+  /**
+   * Returns the String in question but with all characters in upper case.
+   * @return {String}
+   */
+
+  toUpper() {
+    const characters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
NITPICK: This code is duplicated in line 19. You can create an object property to hold this value.